src/main.js

```diff
@@ -84,7 +92,20 @@ export const start = async () => {
     // if the user is requesting a different tenant (indicated by the URL),
     // we swap out these tokens later
     loading = true
-    const tokens = await login()
+
+    let tokens
+
+    // We check local storage for a fallback token and compare it against
+    // the curent timestamp - 1 minute. If it falls outside that window
+    // we attempt the normal flow
+    const fallbackTimestamp = localStorage.getItem('prefect_fallback_auth')
```

Comment:
Are we clearing localStorage anywhere? Would session storage (which clears after each session) be a better fit here? (I guess it doesn't matter as we check date but may slow things down a little to call timeout twice?)